core/templates/pages/certificate-assessment-player-page/certificate-assessment-conversation-skin.component.css

```
@@ -0,0 +1,169 @@
+    background-color: white;
+    border-radius: 4px;
+    box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+    margin: 0 20px;
+    padding: 20px 20px 16px;
+  }
+
+  .certificate-assessment-progress-meta {
+    color: #334e68;
+    align-items: center;
+    display: flex;
+    font-family: 'Roboto', sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)